src/cli/cli_tcp.cpp

```diff
@@ -85,6 +85,31 @@ void TcpExample::MbedTlsDebugOutput(void *ctx, int level, const char *file, int
 }
 #endif // OPENTHREAD_CONFIG_TLS_ENABLE
 
+/**
+ * @cli tcp init
+ * @code
+ * tcp init tls
+ * Done
+ * @endcode
+ * @cparam tcp init [@ca{mode}] [@ca{size}]
+ * * The `mode` has three possible values:
+ *   * `tls`: Specifies that the TCP connection between two nodes should also
+ *     use the TLS protocol on top of TCP. When two nodes communicate over TCP,
+ *     both nodes must either use TLS or neither node should use TLS because
+ *     a non-TLS endpoint cannot communicate with a TLS endpoint.
+ *   * `linked` or `circular`: Either one of these options means that TLS
+ *     is not to be used, and the specified buffering type should be used for TCP
+ *     buffering. The behavior of `linked` and `circular` is identical. Examine the code
+ *     for the differences between these two buffering types.
+ *     Two endpoints of a TCP connection are not required to use the same buffering type.
+ * * The `size` parameter sets the size of the receive buffer to associate with the
+ *   example TCP endpoint. If left unspecified, the maximum size is used.
```

Comment:
Is the maximum size defined somewhere?

Reply:
@abtink Plz see Jeff's question here, thanks.

Reply:
Specified by `OPENTHREAD_CONFIG_CLI_TCP_RECEIVE_BUFFER_SIZE` config option.

Reply:
thx!